queue_services/entity-filer/src/entity_filer/filing_processors/alteration.py

```diff
@@ -92,11 +92,16 @@ def process(
     # update rules, if any
     with suppress(IndexError, KeyError, TypeError):
         rules_file_key = dpath.util.get(filing, '/alteration/rulesFileKey')
-        rules_and_memorandum.update_rules(business, filing_submission, rules_file_key)
+        rules_file_name = dpath.util.get(filing, '/alteration/rulesFileName')
```

Comment:
Ethan is back on Monday - I'll let him know - Lol sorry about the file name issues Vysakh

Reply:
@ethantspitt -^

Reply:
@ethantspitt Are we planning to do the same thing in incorporation application and/or correction? Is it applicable only for rule file?

Reply:
For COOP corrections - yes, and it's only applicable for rules with corrections because user can only upload a new rules for special resolution.

For COOP incorporation application - probably yes for both rules and memorandum to keep consistency, I'll check this with Janis.